vue/src/TagmanagerTrackingCode/TrackingSPAPage.vue

```diff
@@ -0,0 +1,186 @@
+<!--
+  Matomo - free/libre analytics platform
+  @link https://matomo.org
+  @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+-->
+<template>
+  <ol class="list-style-decimal">
+    <li v-html="$sanitize(setupStep1)"></li>
+    <TrackingCodeCommon
+      :show-container-row="showContainerRow"
+      :current-action="currentAction"
+      :showBottom="false"
+      :showDescription="false"
+      :showStep2="false"
+      :showStep3="false"
+      @fetchInstallInstructions="fetchInstallInstructionsSPA"
+      ref="trackingCodeCommon"
+    />
+    <li v-html="$sanitize(setupStep2)"></li>
+    <li v-html="$sanitize(setupStep3)"></li>
+    <li v-text="translate('TagManager_SPAFollowStep4')"></li>
+    <li v-text="translate('TagManager_SPAFollowStep5')"></li>
+    <li v-html="$sanitize(fetchFollowStepCommon(5))"></li>
```

Comment:
I think this is supposed to be `6`? When I test locally, it's showing the same text for steps 5 and 6.

Reply:
@snake14 Updated points now point 4 and 6 is same as we have same header for both the buttons